OpenRA.Game/Network/Order.cs

```diff
@@ -147,16 +152,24 @@ public static Order Deserialize(World world, BinaryReader r)
 
 						var targetString = flags.HasField(OrderFields.TargetString) ? r.ReadString() : null;
 						var queued = flags.HasField(OrderFields.Queued);
+
+						List<Actor> extraActors = null;
+						if (flags.HasField(OrderFields.ExtraActors) && world != null)
```

Comment:
If `flags.HasField(OrderFields.ExtraActors)` is true but `world != null`  is false, are you sure you don't want to consume the extra bytes?

Reply:
Fixed.